src/dotnet/CommonLocalizableStrings.cs

```diff
@@ -5,183 +5,65 @@ namespace Microsoft.DotNet.Tools
 {
     internal class CommonLocalizableStrings
     {
-        public const string UnsupportedProjectType = "Unsupported project type. Please check with your sdk provider.";
         public const string ProjectAlreadyHasAreference = "Project already has a reference to `{0}`.";
         public const string ProjectReferenceCouldNotBeFound = "Project reference `{0}` could not be found.";
         public const string ProjectReferenceRemoved = "Project reference `{0}` removed.";
 
         // Project related
-        public const string Project = "Project";
```

Comment:
UnsupportedProjectType will be needed once we have the ProjectTypeGuid lookup in place. @jonsequitur we added many of these strings proactively because the turnaround time for Loc is non-0. Can we move them to a separate section of the file instead of deleting?